src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1008,6 +1031,7 @@ void
 PeerImp::onMessageUnknown(std::uint16_t type)
 {
     // TODO
+    JLOG(journal_.debug()) << "onMessageUnknown: " << type;
```

Comment:
This `JLOG` can be used to flood logs and provided little useful information. I'd remove it.

Reply:
Done, removed.